.github/workflows/pr-build.yml

```diff
@@ -0,0 +1,85 @@
+name: PR Build
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build-artifacts:
+    runs-on: ubuntu-20.04
+    name: Build artifacts - ${{ matrix.name }}
+    strategy:
+      matrix:
+        include:
+          - language: java
+            build_directory: opentelemetry-lambda/java
+            build_command: ./gradlew build
+          - language: nodejs
+            build_directory: opentelemetry-lambda/nodejs
+            build_command: npm install
+          - language: python
```

Comment:
I'm worried python will be flaky because it uses public ECR

Reply:
any reason not use docker image `amazon/aws-sam-cli-build-image-python3.8`?

Reply:
Docker hub also has low anonymous pull rate. My workaround in upstream repos is to copy to GHCR, here we'd usually copy to ECR, but doesn't help...